src/ReactTestWrapper.js

```diff
@@ -29,7 +29,8 @@ export default class ReactTestWrapper extends TestWrapper {
   }
 
   attr (name) {
-    return this.el.getAttribute(name) || undefined
+    const value = this.el.getAttribute(name)
+    return value !== null ? value : undefined
```

Comment:
that seems right to me. note that "empty attribute" and "empty string attribute" and "null attribute" and "undefined attribute" all need to be treated differently.